metrics/heapster.go

```diff
@@ -54,6 +53,7 @@ var (
 	argAllowedUsers     = flag.String("allowed_users", "", "comma-separated list of allowed users")
 	argSources          flags.Uris
 	argSinks            flags.Uris
+	argProcessors       = flag.String("processors", "kubernetes", "processors for heapster")
```

Comment:
Actually, lets name the default set "default".  As Heapster works only in kubernetes name "kubernetes" doesn't mean anything.